easybuild/easyconfigs/b/Beast/Beast-1.8.4-foss-2016a.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'Beast'
+version = '1.8.4'
+
+homepage = 'http://beast.bio.ed.ac.uk/'
+description = """ BEAST is a cross-platform program for Bayesian MCMC analysis of molecular 
+ sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using 
+ strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies 
+ but is also a framework for testing evolutionary hypotheses without conditioning on a single 
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```

Comment:
@ysagon using a 'proper' toolchain doesn't make much sense in this case, since nothing is actually being compiled? also, none of the dependencies were installed with a non-dummy toolchain either

so, let's change this to:

```python
toolchain = {'name': 'dummy', 'version': ''}
```

You will need to change the easyconfig filename accordingly.

Also, please include an empty line in between the `toolchain = ` and the `source_urls` line.